src/lib/updating-element.ts

```diff
@@ -433,7 +433,7 @@ export abstract class UpdatingElement extends HTMLElement {
    * Returns the property name for the given attribute `name`.
    * @nocollapse
    */
-  private static _attributeNameForProperty(
+  protected static _attributeNameForProperty(
```

Comment:
Please remove the leading underscore. Our convention is that this should only be used for private properties.